### PR TITLE
Updated commands and added module for colorful printable output.

### PR DESCRIPTION
**Pull request by mayurilahane**
- Updated commands by removing "pass"
- For error messages, you will get the green-colored message.
please have a look @ganeshhubale  @Yadnyawalkya

### CODE REVIEW

File: pygenpass/database.py [hunk 1]
```diff
@@ -40,18 +42,23 @@ def __init__(self):
...
+                """INSERT INTO passwords
+                (portal_name, password, creation_date, email, portal_url)
+                VALUES (?, ?, ?, ?, ?)""",
+                (self.portal_name, self.password, self.creation_date, self.email, self.portal_url),
+            )
+            self.con.commit()
+        except sqlite3.IntegrityError:
+            print(
+                colored("Already exists with the same name. Try with another Portal_name", "green")
+            )
```
Comment (ganeshhubale) [inline pygenpass/database.py:60]: :100:

File: pygenpass/database.py [hunk 2]
```diff
@@ -40,18 +42,23 @@ def __init__(self):
...
-            (self.portal_name, self.password, self.creation_date, self.email, self.portal_url),
-        )
-        self.con.commit()
+        try:
+            self.password = password
+            self.creation_date = creation_date
+            self.email = email
+            self.portal_name = portal_name
+            self.portal_url = portal_url
+            self.cursor_obj.execute(
```
Comment (ganeshhubale) [inline pygenpass/database.py:51]: @mayurilahane Why do we need variable initialization inside the try block?

**Review by ganeshhubale — CHANGES_REQUESTED** [review]

**Comment by ganeshhubale** [comment]
Any updates on this PR @mayurilahane ?

File: pygenpass/database.py (hunk 2 above)
Comment (mayurilahane) [reply pygenpass/database.py:51]: yes, agree!
we can do this try where we are actually getting values.
thanks for suggesting!

**Pull request merged**